### DISCUSSION TITLE
Federation in Kanidm ?

### DISCUSSION THREAD

**Discussion by htcosta** [kanidm/kanidm#2455] [Q&A] [answered]
Does kanidm support any kind of brokered federation ?

**Comment by yaleman** [comment, marked as answer]
No, the product is an IDP rather than a federation system - we're looking at one day supporting trust between instances.

**Comment by Firstyear** [comment]
Can you explain what you mean by "brokered federation" here? I think that the term is too vague for us to know what it is you mean.
